plugins/modules/aci_vrf_leak_internal_subnet.py

```diff
@@ -0,0 +1,397 @@
+options:
+  tenant:
+    description:
+    - The name of the Tenant the VRF belongs to.
+    type: str
+    aliases: [ tenant_name ]
+  vrf:
+    description:
+    - The name of the VRF.
+    type: str
+    aliases: [ context, name, vrf_name ]
+  description:
+    description:
+    - The description for the VRF Leak Internal Subnet.
+    type: str
+    aliases: [ descr ]
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+  scope:
+    description:
+    - Scope of the object.
+    type: str
+    choices: [ public, private, shared ]
+    default: private
+  leak_to:
+    description:
+    - The VRFs to leak the subnet routes into.
+    type: list
+    elements: dict
+    suboptions:
+      tenant:
+        description:
+        - Name of the tenant.
+        type: str
+        aliases: [ tenant_name ]
+      vrf:
+        description:
+        - Name of the VRF.
+        type: str
+        aliases: [ vrf_name ]
+  ip:
+    description:
+    - The IP address / subnet used to match routes to be leaked.
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+notes:
+- The C(tenant) and C(vrf) used must exist before using this module in your playbook.
+  The M(cisco.aci.aci_tenant) and M(cisco.aci.aci_vrf) modules can be used for this.
+seealso:
+- module: cisco.aci.aci_tenant
+- module: cisco.aci.aci_vrf
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(leak:InternalSubnet).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Abraham Mughal (@abmughal)
+"""
+
+EXAMPLES = r"""
+- name: Create leak internal subnet
+  cisco.aci.aci_vrf_leak_internal_subnet:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    vrf: vrf_lab
+    tenant: lab_tenant
+    descr: Lab VRF
+    state: present
+    leak_to:
+      - vrf: "test"
+        tenant: "lab_tenant"
+      - vrf: "test2"
+        tenant: "lab_tenant"
+    description: Ansible Test
+    ip: 1.1.1.2
+  delegate_to: localhost
+
+- name: Remove a subnet from leaking
+  cisco.aci.aci_vrf_leak_internal_subnet:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    vrf: vrf_lab
+    tenant: lab_tenant
+    state: absent
+    leak_to:
+      - vrf: "test2"
+        tenant: "lab_tenant"
+    description: Ansible Test
+    ip: 1.1.1.2
+  delegate_to: localhost
+
+- name: Delete leak internal subnet
+  cisco.aci.aci_vrf_leak_internal_subnet:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    vrf: vrf_lab
+    tenant: lab_tenant
+    state: absent
+    description: Ansible Test
+    ip: 1.1.1.2
```

Comment:
In the bottom you say leak_to is required when absent (line 294) but there is not leak_to here